plugins/woocommerce/tests/performance/requests/merchant/add-order.js

```diff
@@ -116,7 +113,7 @@ export function addOrder( includeTests = {} ) {
 		check( response, {
 			'is status 200': ( r ) => r.status === 200,
 			"body contains: 'Add new order' header": ( response ) =>
-				response.body.includes( `${ admin_new_order_assert }` ),
+				response.body.includes( 'Add new order' ),
```

Comment:
Yup. So in `trunk`, the value of `admin_new_order_assert`  points to the HTML element with the attributes `post_status" type="hidden" value="pending` when HPOS is enabled. However, this element no longer exists when HPOS is enabled, so this test will always fail without this correction.

On the other hand, `Add new order` appears in both cases (when HPOS enabled/disabled) as a header. So I thought maybe we don't need to have a conditional value for the header anymore, and therefore we could hardcode it just like in other parts of the test.

With that said, I think I should have used `Add new order</h1>` which was the original value of `admin_new_order_assert` in trunk when HPOS is off, instead of simply `Add new order`, because the assertion is about verifying the "Add new order" header. I'll push a commit to change that shortly.

Reply:
> With that said, I think I should have used `Add new order</h1> `

Oops, bad idea to add the  `</h1>` closing tag 😓 It's because when HPOS is enabled, the header appears as `Add new order		</h1>` (with the huge amount of whitespace). I reverted adding the `</h>` closing tag.